### DISCUSSION TITLE
Turning Lights on and off

### DISCUSSION THREAD

**Discussion by stuartHBGaming** [jnsmalm/pixi3d#165] [Q&A] [answered]
Hi, 

I have been trying to see if there is way of turning off lights when required.

Is it just a case of removing the light from the LightingEnvironment.main.lights array or is there some method on the actual light where you can control on and off?

**Comment by jnsmalm** [comment, marked as answer]
Hi! Yes, right now you need to add/remove lights from the lighting environment to control on/off.

**Reply by stuartHBGaming** [reply]
thats fab, thank you very much
